app/gui/src/model/undo_redo.rs

```diff
@@ -103,15 +109,7 @@ impl Transaction {
     /// the current transaction in context where it is not clear whether transaction was already set
     /// up or not.
     pub fn fill_content(&self, id: model::module::Id, content: model::module::Content) {
-        with(self.frame.borrow_mut(), |mut data| {
-            debug!(
-                "Filling transaction '{}' with snapshot of module '{id}':\n{content}",
-                data.name
-            );
-            if data.snapshots.try_insert(id, content).is_err() {
-                debug!("Skipping this snapshot, as module's state was already saved.")
-            }
-        })
+        with(self.frame.borrow_mut(), |mut data| data.store_module_content(id, content))
```

Comment:
you can use `self.frame.with_borrowed_mut(|...| ...)` instead.

Reply:
@wdanilo 
As far as I know, there is no such method on `RefCell`, it seems to be provided only for things defined with `thread_local`.

Reply:
This method is defined in prelude!